Sources/HeliumLogger/HeliumLogger.swift

```diff
@@ -61,16 +61,133 @@ public class HeliumLogger {
     /// If true, use the detailed format when a user logging format wasn't specified.
     public var details: Bool = true
 
+    /// If true, use the full file path, not just the filename.
+    public var fullFilePath: Bool = false
```

Comment:
This seems more consistent with the current enums being completely distinct entities that can be more finely tuned using other object states - like `(%date)` in conjunction with `dateFormat` rather than `(%datetime)` `(%date)` and `(%time)`. 

But I can see the benefit of doing it the way you suggested too.